dotcom-rendering/src/components/InlineProductCard.tsx

```diff
@@ -0,0 +1,215 @@
+import { palette } from '../palette';
+import type { ProductCta, ProductImage } from '../types/content';
+import { ProductCardButtons } from './ProductCardButtons';
+import { ProductCardImage } from './ProductCardImage';
+
+export type CustomAttributes = {
+	name: string;
+	value: string;
+};
+
+export type InlineProductCardProps = {
+	format: ArticleFormat;
+	brandName: string;
+	productName: string;
+	image?: ProductImage;
+	productCtas: ProductCta[];
+	customAttributes: CustomAttributes[];
+	isCardOnly: boolean;
+	shouldShowLeftColCard?: boolean;
+	lowestPrice?: string;
+};
+
+const baseCard = css`
+	padding: ${space[2]}px ${space[3]}px ${space[3]}px;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	column-gap: 10px;
+	row-gap: ${space[4]}px;
+	max-width: 100%;
+	img {
+		width: 100%;
+		height: auto;
+	}
+	${from.mobileLandscape} {
+		column-gap: 20px;
+	}
+`;
+
+const hideFromWide = css`
+	${from.wide} {
+		display: none;
+	}
+`;
+
+const showcaseCard = css`
+	${baseCard};
+	background-color: ${palette('--product-card-background')};
+	border-top: 1px solid ${palette('--product-card-border')};
+`;
+
+const productCard = css`
+	${baseCard};
+	padding: ${space[2]}px 0 0;
+	background-color: transparent;
+	border-top: 1px solid ${palette('--section-border')};
+`;
+
+const productInfoContainer = css`
+	display: flex;
+	flex-direction: column;
+	gap: ${space[1]}px;
+	${textSans17};
+
+	${from.mobileLandscape} {
+		${textSans20};
+	}
+`;
+
+const primaryHeading = css`
+	${headlineMedium20};
+	${from.mobileLandscape} {
+		${headlineMedium24};
+	}
+`;
+
+const productNameStyle = css`
+	${textSans17};
+	> strong {
+		${textSansBold17}
+	}
+
+	${from.mobileLandscape} {
+		${textSans20};
+		> strong {
+			${textSansBold20}
+		}
+	}
+`;
+
+const mobileButtonWrapper = css`
+	grid-column: 1 / span 2;
+	display: grid;
+	row-gap: ${space[1]}px;
+	${from.mobileLandscape} {
+		display: none;
+	}
+`;
+
+const desktopButtonWrapper = css`
+	display: none;
+	${from.mobileLandscape} {
+		display: grid;
+		row-gap: ${space[1]}px;
+	}
+`;
+
+const customAttributesContainer = css`
+	grid-column: 1 / span 2;
+	border-top: 1px solid ${palette('--product-card-border-neutral')};
+	padding-top: ${space[2]}px;
+	display: grid;
+	gap: ${space[3]}px;
+
+	${from.mobileLandscape} {
+		grid-template-columns: 1fr 1fr;
+		gap: ${space[5]}px;
+	}
+`;
+
+const customAttributeItem = css`
+	${textSans15};
+	${from.phablet} {
+		${textSans17};
+	}
+	strong {
+		font-weight: 700;
+	}
+`;
+
+const CustomAttribute = ({ name, value }: CustomAttributes) => (
+	<div css={customAttributeItem}>
+		<strong>{name}</strong>
+		<br />
+		{value}
+	</div>
+);
+
+export const InlineProductCard = ({
+	format,
+	brandName,
+	productName,
+	image,
+	customAttributes,
+	productCtas,
+	isCardOnly = false,
+	shouldShowLeftColCard = false,
+	lowestPrice,
+}: InlineProductCardProps) => {
+	return (
+		<div
+			css={[
+				isCardOnly && productCard,
+				!isCardOnly && showcaseCard,
```

Comment:
Is this the same as?

```suggestion
				isCardOnly ? productCard : showcaseCard,
```

I think which is preferable for readability may be subjective, so up to you.